src/Workspaces/Core/Portable/Options/OptionChangedEventArgs.cs

```diff
@@ -1,22 +1,22 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#nullable enable
+
 using System;
 
 namespace Microsoft.CodeAnalysis.Options
 {
     internal sealed class OptionChangedEventArgs : EventArgs
     {
-        private readonly OptionKey _optionKey;
-        private readonly object _value;
-
-        internal OptionChangedEventArgs(OptionKey optionKey, object value)
+        internal OptionChangedEventArgs(OptionKey optionKey, object? value)
         {
-            _optionKey = optionKey;
-            _value = value;
+            Option = optionKey.Option;
+            Language = optionKey.Language;
+            Value = value;
         }
 
-        public IOption Option => _optionKey.Option;
-        public string Language => _optionKey.Language;
-        public object Value => _value;
+        public IOption Option { get; }
+        public string? Language { get; }
+        public object? Value { get; }
```

Comment:
Move in front of the ctor, since these properties are now data members. You can also make them readonly fields since they are effectively internal.